PokeDemo/Controllers/PokemonListViewController.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  ViewController.swift
+//  PokeDemo
+//
+//  Created by Rudr Bansal on 05/07/22.
+//
+// MARK: - Navigation
+
+import UIKit
+
+class PokemonListViewController: BaseViewController {
+    
+    // MARK: - IBOutlets
+    
+    @IBOutlet weak var tableView: UITableView!
+    
+    // MARK: - Properties
+    
+    // MARK: - Private
+    
+    private let presenter = PokemonListPresenter()
+    private var pokemons: [Pokemon]? = [Pokemon]()
+    
+    // MARK: - View Life Cycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tableView.register(UINib(nibName: PokemonListCell.cellIdentifier(), bundle: nil), forCellReuseIdentifier: PokemonListCell.cellIdentifier())
+        tableView.dataSource = self
+        tableView.delegate = self
+        presenter.setViewDelegate(delegate: self)
+        presenter.viewDidLoad()
+    }
+}
+
+// MARK: - Extensions
+
+extension PokemonListViewController: UITableViewDataSource {
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return pokemons?.count ?? 0
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: PokemonListCell.cellIdentifier()) as! PokemonListCell
```

Comment:
**issue:** Force unwrap can crash if cellIdentifier is changed. Someone changed cellIdentifier in storyboard.